packages/tests/unit-tests/test-cases/expected/angular/refs-as-attribute-helper.tsx

```diff
@@ -0,0 +1,52 @@
+import { Input } from "@angular/core";
+class HelperWidgetProps {
+  @Input() forwardRef?: HTMLDivElement | null;
+  @Input() someRef?: HTMLDivElement | null;
+  @Input() refProp?: HTMLDivElement | null;
+  @Input() forwardRefProp?: HTMLDivElement | null;
+}
+import {
+  Component,
+  NgModule,
+  ChangeDetectionStrategy,
+  ChangeDetectorRef,
+  ViewRef,
+} from "@angular/core";
+import { CommonModule } from "@angular/common";
+
+@Component({
+  selector: "dx-helper-widget",
+  changeDetection: ChangeDetectionStrategy.OnPush,
+  inputs: ["forwardRef", "someRef", "refProp", "forwardRefProp"],
+  template: `<div
+    ><div>Ref:{{(===undefined||===null?undefined:.someRef)}}</div
+    ><div>ForwardRef:{{(===undefined||===null?undefined:.forwardRef)}}</div
+    ><div>RefProp:{{(===undefined||===null?undefined:.refProp)}}</div
```

Comment:
It' is a strange. Where is a left part of comparing? `===undefined||===null`
I suppose it is because in angulare we have no `props` object